libp2p_playground_test.go

```diff
@@ -260,9 +260,9 @@ func TestBasic(t *testing.T) {
 
 func TestUniDirection(t *testing.T) {
 	glog.Infof("\n\nTest Unidirection...")
-	dht1, h1 := makeRandomHost(10000)
+	dht1, h1 := makeRandomHost(10002)
```

Comment:
Any particular reason for the different port numbers?

Reply:
There is a test above (`TestBasic`) this that uses 10000 and 10001.  Even with the explicit `Close()`, sometimes the ports don't get released quickly enough, that the initiation context times out for `TestUniDirection`.  Not the greatest "fix", but gets tests to pass more consistently.

Reply:
There is a test above (`TestBasic`) this that uses 10000 and 10001.  Even with the explicit `Close()`, sometimes the ports don't get released quickly enough, that the initiation context times out for `TestUniDirection`.  Not the greatest "fix", but gets tests to pass more consistently.